docs/getting-started/siem-migration.asciidoc

```diff
@@ -1,5 +1,142 @@
 [[siem-migration]]
 [chapter]
-= AI-powered SIEM migration
+= Automatic migration
 
-This page is a placeholder 
\ No newline at end of file
+WARNING: This feature is in technical preview. It may change in the future, and you should exercise caution when using it in production environments. Elastic will work to fix any issues, but features in technical preview are not subject to the support SLA of GA features.
+
+Automatic Migration for detection rules helps you quickly convert SIEM rules from the Splunk Processing Language (SPL) to the Elasticsearch Query Language ({esql}). If comparable Elastic-authored rules exist, it simplifies onboarding by mapping your rules to them. Otherwise, it creates custom rules on the fly so you can verify and edit them instead of writing them from scratch.
+
+You can ingest your data before migrating your rules, or migrate your rules first in which case the tool will recommend which data sources you need to power your migrated rules. 
+
+.Requirements
+[sidebar]
+--
+* The `SIEM migrations: All` Security sub-feature privilege.
+* A working <<llm-connector-guides, LLM connector>>.
+* {stack} users: an https://www.elastic.co/pricing[Enterprise] subscription. 
+* {Stack} users: {ml} must be enabled.
+* {serverless-short} users: a https://www.elastic.co/pricing/serverless-security[Security Complete] subscription.
+* {ecloud} users: {ml} must be enabled. We recommend a minimum size of 4GB of RAM per {ml} zone.
+--
+
+[discrete]
+== Get started with Automatic Migration
+
+1. Find **Get started** in the navigation menu or use the {kibana-ref}/introduction.html#kibana-navigation-search[global search field].
+2. Under **Configure AI provider**, select a configured model or <<llm-connector-guides, add a new one>>. For information on how different models perform, refer to the <<llm-performance-matrix>>.
+3. Next, under **Migrate rules & add data**, click **Translate your existing SIEM rules to Elastic**, then **Upload rules**.
+4. Follow the instructions on the **Upload Splunk SIEM rules** flyout to export your rules from Splunk as JSON. 
++
+image::images/security-siem-migration-1.png[automatic migration setup page,700]
++
+.NOTE
+[sidebar] 
+--
+The provided query downloads Splunk correlation rules and saved searches. Alternatively, as long as you    export your results in a JSON format, you can use a different query. For example:
+
+```
```

Comment:
Just checking if you wanted to use a [code block](https://docs.elastic.dev/migration/syntax/code-blocks#basic-code-blocks) (which allows for easy copying) for the following example query.